openshift/template.yaml

```diff
@@ -198,6 +198,9 @@ parameters:
 - name: INSTALLER_CACHE_CAPACITY
   value: "6442450944"
   required: false
+- name: ORPHANED_EVENTS_MAX_AGE
+  required: false
+  value: "72h"
```

Comment:
did you mean 720h?

Reply:
I actually meant to override it to 72h, but now that you mention it it would be good to have consistency at least with the starting value.
I believe this is pretty safe to delete after 72h as only records that did not cascade delete would be here, and l would set it by default in the code as well. WDYT?

Reply:
sgtm

Reply:
@gamli75 WDYT? given what we found out, is 3 days enough?

Reply:
Do these events have any use?

Reply:
They are pretty much unusable from the application perspective, as they will never be able to be retrieved via API until a cluster with the same cluster ID exists.
They are also collected by event collection system AKA "kibana"
AFAIU their only use would be diagnostic

Reply:
The question is if we should just not send the event.  WDYT?

Reply:
I think the event is giving good insight for people investigating the cluster, and would give good feedback to the user. Example messages:
`Failed to register cluster. Error: OpenShiftSDN network type is not allowed in single node mode`

`Failed to register cluster. Error: Invalid format for Tags: tag-3. Tags should be a comma-separated list (e.g. tag1,tag2,tag3). Each tag can consist of the following characters: Alphanumeric (aA-zZ, 0-9), underscore (_) and white-spaces.`

`Failed to register cluster. Error: Can't set oci platform on s390x architecture`

If anything, I would try to show it to the user. For now this is not possible because the cluster is not registered, and we can't retrieve events from non-registered clusters (due to security measure to avoid cross-tenant access) - chicken and egg problem.

Reply:
Fair enough, I agree we can probably remove the event generation and clean up in a different way.